x/leverage/simulation/operations_test.go

```diff
@@ -29,15 +29,32 @@ type SimTestSuite struct {
 // SetupTest creates a new umee base app
 func (s *SimTestSuite) SetupTest() {
 	checkTx := false
-	s.app = umeeappbeta.Setup(s.T(), checkTx, 1)
-	s.ctx = s.app.BaseApp.NewContext(checkTx, tmproto.Header{
-		ChainID: fmt.Sprintf("test-chain-%s", tmrand.Str(4)),
-		Height:  1,
-	})
+	betaApp := umeeappbeta.Setup(s.T(), checkTx, 1)
+	ctx := betaApp.NewContext(checkTx, tmproto.Header{})
+
+	// Note: Setting umee collateral weight to 1.0 to allow lender to borrow heavily
+	umeeToken := types.Token{
```

Comment:
Let's create three different accepted assets and give them different parameters and oracle prices, for maximum chaos.

(Notably, so the simulation can test liquidations where the repay and reward denom are different, and other such scenarios)

Reply:
done at https://github.com/umee-network/umee/pull/433/commits/fe4fa741d42ca44780be0911235763b174e3bf4f